src/main/asciidoc/jdbc.adoc

```diff
@@ -371,21 +371,21 @@ The following table describes the available events:
 |===
 | Event | When It Is Published
 
-| {javadoc-base}org/springframework/data/relational/core/mapping/event/BeforeDeleteEvent.java[`BeforeDeleteEvent`]
+| {javadoc-base}org/springframework/data/relational/core/mapping/event/BeforeDeleteEvent.html[`BeforeDeleteEvent`]
 | Before an aggregate root gets deleted.
 
-| {javadoc-base}org/springframework/data/relational/core/mapping/event/AfterDeleteEvent.java[`AfterDeleteEvent`]
+| {javadoc-base}org/springframework/data/relational/core/mapping/event/AfterDeleteEvent.html[`AfterDeleteEvent`]
 | After an aggregate root gets deleted.
 
-| {javadoc-base}/org/springframework/data/relational/core/mapping/event/AfterDelete.java[`BeforeDeleteEvent`]
+| {javadoc-base}/org/springframework/data/relational/core/mapping/event/AfterDelete.html[`BeforeSaveEvent`]
```

Comment:
I think you have a typo. Looking at [the event folder](https://github.com/spring-projects/spring-data-jdbc/tree/master/src/main/java/org/springframework/data/relational/core/mapping/event) you probably want: ```BeforeSaveEvent.html[`BeforeSaveEvent`]```

Reply:
You are absolutely right, will fix this right away.